scripts/install/msys64_installer.sh

```diff
@@ -20,6 +20,7 @@ declare -a BASE_PACKAGES=(
 )
 
 declare -a OPTIONAL_PACKAGES=(
+  "git"                       # Distribution script (check_submodules_update.sh)
```

Comment:
Is that really needed? Git should be anyhow installed, otherwise the checkout of the repo is not possible.

Reply:
The git dependency is not related to the ROS fix, but I found some errors while reviewing the logs:

> * `git` is added to the MSYS2 dependencies because (https://github.com/cyberbotics/webots/runs/2888587906):
>   ```
>   # removing files and folders not in git repository
>   /bin/sh: line 1: git: command not found
>   # generating project files
>   # updating proto cache
>   # checking submodules update
>   ./check_submodules_update.sh: line 29: git: command not found
>   ./check_submodules_update.sh: line 30: git: command not found
>   ./check_submodules_update.sh: line 33: git: command not found
>   ```

— https://github.com/cyberbotics/webots/pull/3173#issuecomment-866570817

It is not necessary to include, the CI doesn't fail, it just displays the warnings

Reply:
That is probably because the original git is not the one of MSYS2 and cannot be found from the MSYS2 console due to a missing PATH. Adding it here doesn't hurt too much as it's not a big download.